src/drivers/vmount/input_mavlink.cpp

```diff
@@ -285,7 +285,7 @@ int InputMavlinkCmdMount::update_impl(unsigned int timeout_ms, ControlData **con
 
 void InputMavlinkCmdMount::_ack_vehicle_command(uint16_t command)
 {
-	vehicle_command_ack_s vehicle_command_ack;
+	vehicle_command_ack_s vehicle_command_ack = {};
```

Comment:
As in further patch in this PR a new field is added to vehicle_command_ack I choose to zeroing all vehicle_command_ack to make it safe but I can just set this field in all vehicle_command_ack.